Modules/Tracker/LinePool.lua

```diff
@@ -213,130 +259,242 @@ function LinePool.Initialize(trackedQuestsFrame, UntrackQuest, TrackerUpdate)
 
         line.expandQuest = expandQuest
 
+        if Questie.IsWotlk then
+            line:HookScript("OnUpdate", line.OnUpdate)
+        end
+
         linePool[i] = line
         lastFrame = line
     end
-end
 
----@param trackedAchievementsFrame Frame
-function LinePool.InitializeAchievementLines(trackedAchievementsFrame, OnAchievementClick, AchievementUpdate)
-    local trackerFontSizeQuest = Questie.db.global.trackerFontSizeQuest
+    -- create buttonPool for quest items
+    for i = 1, C_QuestLog.GetMaxNumQuestsCanAccept() do
+        local buttonName = "Questie_ItemButton"..i
+        local btn = CreateFrame("Button", buttonName, UIParent, "SecureActionButtonTemplate")
+        local cooldown = CreateFrame("Cooldown", nil, btn, "CooldownFrameTemplate")
+        btn.range = btn:CreateFontString(nil, "OVERLAY", "NumberFontNormalSmallGray")
+        btn.count = btn:CreateFontString(nil, "ARTWORK", "Game10Font_o1")
+        btn:SetAttribute("type1", "item")
+        btn:SetAttribute("type2", "stop")
+        btn:Hide()
+
+        if Questie.db.global.trackerFadeQuestItemButtons then
+            btn:SetAlpha(0)
+        end
 
-    local lastFrame
-    for i = 1, achievementPoolSize do
-        local line = CreateFrame("Button", nil, trackedAchievementsFrame)
-        line:SetSize(1, 1)
+        btn.SetItem = function(self, quest, buttonType, size)
```

Comment:
Do you think it would be clever to move this function somewhere else as it is quite big?